packages/lit-html/src/lit-html.ts

```diff
@@ -438,12 +438,16 @@ if (ENABLE_EXTRA_SECURITY_HOOKS) {
   }
 }
 
-const walker = d.createTreeWalker(
-  d,
-  129 /* NodeFilter.SHOW_{ELEMENT|COMMENT} */,
-  null,
-  false
-);
+const walker = (
+  d.createTreeWalker as (
+    root: Node,
+    whatToShow: number,
+    filter: NodeFilter | null,
+    // All 4 parameters are required for IE11, but TypeScript 4.4 removed the
+    // entityReferenceExpansion parameter. This is the previous signature.
+    entityReferenceExpansion?: boolean
+  ) => TreeWalker
+)(d, 129 /* NodeFilter.SHOW_{ELEMENT|COMMENT} */, null, false);
```

Comment:
This seems awkward in comparison to just depending on the IE11 types.

Another nice part about depending on IE11 types is that if/when we drop IE11 support, we can drop those types and the error will let us know we can drop this hack

Reply:
True, done.

We will need to add this dependency, or write a patch, when we next import into google3, though.

Reply:
PTAL @rictic